src/Compilers/CSharp/Portable/CodeGen/Optimizer.cs

```diff
@@ -1447,6 +1466,14 @@ public override BoundNode VisitSwitchDispatch(BoundSwitchDispatch node)
             return node.Update(boundExpression, node.Cases, node.DefaultLabel, node.LengthBasedStringSwitchDataOpt);
         }
 
+        public override BoundNode VisitLoweredIsPatternExpression(BoundLoweredIsPatternExpression node)
+        {
+            var statements = VisitSideEffects(node.Statements);
+            RecordLabel(node.WhenTrueLabel);
```

Comment:
>RecordLabel [](http://example.com/codeflow?start=12&length=11)

Is this change made in an attempt to fix the "Repro" test?

Reply:
No, only the change referenced in PR description is related to that issue.
(I've added some context on this one a few comments earlier.)

Reply:
This change makes sense to me. Effectively we have  implicit label statements for our labels and when we visit a ```BoundLabelStatement```, we call ```RecordLabel```